### ISSUE TITLE
Add output to a text file with full css paths to affected divs.

### ISSUE THREAD

**Issue by tschallacka** [workeffortwaste/layout-shift-gif#11]
For analyzing which divs specifically are affected it would be helpful if a full css selector to the div that has changed could be included in the output.

Something like `body > div > div.foobar > div > span:nth-child(10)`
It makes finding the affected elements much easier.

**Comment by kupietools** [comment]
Yeah, this is not really useful without that. It makes a very pretty gif of flashing green lines but I have no clue what selectors they are so it's not much good.

**Comment by workeffortwaste** [comment]
This is now available in version 1.4.0 of the CLI tool as a separate JSON report saved alongside the image.
